assets/inbuilt/transformers/dockerfile/net5app/templates/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#   Copyright IBM Corporation 2020
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS buildImg
```

Comment:
Can we standardise in our images how we refer to image names? Will it be buildImg or something else?

Reply:
Yes sure, I have changed it to `build_base` for all the images, which we have in other dockerfiles like [javamaven](https://github.com/konveyor/move2kube/blob/a30eb2586e1a31f873820e2b1a3002319e95a95d/assets/inbuilt/transformers/dockerfile/javamaven/templates/Dockerfile#L16).